src/OpenLoco/Graphics/ImageIds.h

```diff
@@ -20,6 +20,8 @@ namespace OpenLoco::ImageIds
 
     constexpr uint32_t close_button = 2321;
 
+    constexpr uint32_t frame_background_image = 2322;
```

Comment:
we like to close up gaps in the list when we know all of the names so could you remove the newline between close_button and this.